aquarius/ddo_checker/shacl_schemas/v4/remote_4.0.0.ttl

```diff
@@ -0,0 +1,351 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+schema:DDOShape
+    sh:targetClass schema:DDO ;
+    sh:property [
+        sh:path schema:id ;
+        sh:datatype xsd:string ;
+        sh:pattern "^did\\:op\\:(.*)$" ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path schema:version ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path schema:chainId;
+        sh:datatype xsd:integer ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path schema:metadata ;
+        sh:node schema:MetadataShape ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:services ;
+        sh:node schema:ServiceShape ;
+        sh:minCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:credentials;
+        sh:node schema:CredentialsShape ;
+    ] ; .
+
+
+schema:MetadataShape
+    sh:targetClass schema:Metadata ;
+    sh:property [
+        sh:path schema:description ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.|\\s)*$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:copyrightHolder;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.|\\s)*$" ;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:name ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.|\\s)*$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:type;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.|\\s)*$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:author;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.|\\s)*$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:license;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.|\\s)*$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:links;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.|\\s)*$" ;
```

Comment:
Am I correct that the omission of the `minCount` and `maxCount` means it can be an array?

Reply:
yes, "links" is an array (or if it is one item, we'll interpret it as an array of one element, see the Slack discussions on that)